src/filter_tree/filter_tree.c

```diff
@@ -273,8 +273,8 @@ void _FilterTree_CollectAttributes(const FT_FilterNode *root, rax *attributes) {
 		AR_EXP_CollectAttributes(root->pred.rhs, attributes);
 		break;
 	}
-	default: {
-		assert(0);
+	case FT_N_EXP: {
+		AR_EXP_CollectAttributes(root->exp.exp, attributes);
 		break;
 	}
```

Comment:
Please keep the default in addition to `case FT_N_EXP`.

Reply:
done